config/PLACEHOLDER/secrets_env.yml.PLACEHOLDER

```diff
@@ -0,0 +1,96 @@
+---
+
+
+# A possible way to generate a random salt is by running the following command:
+# LC_CTYPE=C tr -c -d '0123456789abcdefghijklmnopqrstuvwxyz' </dev/urandom | dd bs=15 count=1 2>/dev/null;echo
+# Attention: The secret key should not be more than 16 characters.
+
+vault_ssp_secretsalt: CHANGEME
+vault_ssp_adminpassword: CHANGEME
+vault_ssp_adminpassword_salt: CHANGEME
+vault_ssp_mod_cron_secret: CHANGEME
```

Comment:
The "vault_" prefix naming convention is used when storing vars in an encrypted vault file so I'm not sure we need it here